libraries/botframework-schema/src/teams/index.ts

```diff
@@ -1241,12 +1406,12 @@ export interface TaskModuleTaskInfo {
      * @member {any} [height] This can be a number, representing the task
      * module's height in pixels, or a string, one of: small, medium, large.
      */
-    height?: any;
+    height?: unknown;
```

Comment:
Is this right use of `unknown`? My interpretation is that `unknown` is used when the type is going to be determined later in the application's code. That doesn't seem to fit in this scenario.

We could use a type:

```ts
export type TeamsTaskModuleSize = 'small' | 'medium' | 'large';

export interface TaskModuleTaskInfo {
  /**
   * @member {any} [height] This can be a number, representing the task
   * module's height in pixels, or a string, one of: small, medium, large.
   */
  height?: number | TeamsTaskModuleSize;

  /**
   * @member {any} [width] This can be a number, representing the task module's
   * width in pixels, or a string, one of: small, medium, large.
   */
  width?: number | TeamsTaskModuleSize;
}
```

Or opposite to my earlier feedback do an inline type:
```ts
export interface TaskModuleTaskInfo {
  /**
   * @member {any} [height] This can be a number, representing the task
   * module's height in pixels, or a string, one of: small, medium, large.
   */
  height?: number | 'small' | 'medium' | 'large';

  /**
   * @member {any} [width] This can be a number, representing the task module's
   * width in pixels, or a string, one of: small, medium, large.
   */
  width?: number | 'small' | 'medium' | 'large';
}
```

Reply:
@joshgummersall I've made the changes recommended in Steven's code blocks. How do you feel about the `any` vs. `unknown` part?

Reply:
Using `any` is _almost_ always bad (IMO). It is like saying, "hey, compiler; this type isn't just anything, it's everything, so stop helping me." This is rarely the case; in practice, you should be telling the compiler, "this type is something, but I don't know what, so make sure that I type check it at runtime".

Concerning his suggestions around expressing `height` as something more specific than `unknown`/`any`, he's spot on (as long as that's what the API returns). I'm generally in favor of being as specific as possible with types, rather than overly permissive. Naturally, there are counterexamples, but that is my intuition.